app/components/change-event-type-select.js

```diff
@@ -70,6 +71,20 @@ export default class ChangeEventTypeSelectComponent extends Component {
     ) {
       types = types.filter((t) => this.isIdInList(t.id, CHANGE_EVENTS_PZ_HPZ));
     }
+    if (
+      classification.id == CLASSIFICATION_CODE.WELZIJNSVERENIGING ||
+      classification.id == CLASSIFICATION_CODE.AUTONOME_VERZORGINGSINSTELLING
+      // ||
```

Comment:
I wouldn't comment out code, if we don't need it just delete it

Reply:
In most cases I would agree. But this code will be needed when we onboard the private OCMW associations in the near future. You will notice there are similar bits of commented code at other places.

Some background information, initially the goal was to onboard both public and private OCMW associations at the same time. But the latter are _registered organisations_ and not _administrative units_. In its current state the frontend requires more in-depth changes to properly handle non-administrative unit organisations (a refactoring that is also discussed in https://github.com/lblod/frontend-organization-portal/pull/530). Therefore, we decided to onboard OCMW associations in two phases: now the public ones, and the private ones after refactoring the frontend. At the time of this decision I had already added some code that deals with private OCMW associations. To me it made more sense to temporarily comment out this code, compared to removing it only to re-add it latter on.